app/helpers/wishlist_helper.rb

```diff
@@ -1,2 +1,13 @@
 module WishlistHelper
+  def add_remove_link(wishlist_course_ids, course)
+    link_to "#{wishlist_course_ids.include?(course.id) ? 'Remove from' : 'Add to'} my wishlist",
+            wishlist_path(course_id: course.id, url: request.original_url),
+            method: wishlist_course_ids.include?(course.id) ? :delete : :post
+  end
```

Comment:
Well it was never always a remove link. It's either an add link or a remove link depending on whether the course is in the wishlist already.